controllers/configuration_controller.go

```diff
@@ -242,6 +245,14 @@ func initTFConfigurationMeta(req ctrl.Request, configuration v1beta2.Configurati
 		DestroyJobName:      req.Name + "-" + string(TerraformDestroy),
 	}
 
+	jobNodeSelectorStr := os.Getenv("JOB_NODE_SELECTOR")
```

Comment:
Great job!
We also need to expose the option to the chart like https://github.com/oam-dev/terraform-controller/blob/master/chart/values.yaml#L12-L18.

Reply:
👌